doc/user-guide/profiling/memory.md

```diff
@@ -15,7 +15,7 @@ kernelspec:
 
 
 ```{versionadded} 0.0.18
-[`execute_notebook`](../../api/api.html#execute-notebook)
+`execute_notebook`
```

Comment:
this is the problem we encountered where we can no longer link to sections in rst files, right? can you open an issue in this repo so we don't forget to fix it?

Reply:
Yes, I was looking for solutions but couldn't find any. Will do